community/vault/vault.initd

```diff
@@ -1,21 +1,33 @@
 #!/sbin/openrc-run
-
-supervisor=supervise-daemon
-
 name="Vault server"
 description="Vault is a tool for securely accessing secrets"
+description_reload="Reload configuration"
 
-VAULT_LOG_FILE="/var/log/${RC_SVCNAME}.log"
+extra_started_commands="reload"
 
-command=/usr/sbin/${RC_SVCNAME}
+command="/usr/sbin/${RC_SVCNAME}"
 command_args="${vault_opts}"
 command_user="${RC_SVCNAME}:${RC_SVCNAME}"
 
-start_pre() {
-	checkpath -f -m 0644 -o "$command_user" "$VAULT_LOG_FILE"
-}
+supervisor=supervise-daemon
+pidfile="/run/${RC_SVCNAME}.pid"
```

Comment:
don't think pidfiles matter for supervise-daemon

Reply:
Apparently supervise-daemon will create a default PID file (`/run/supervise-vault.pid` in this case). Will update the PR.